__tests__/app.test.js

```diff
@@ -93,4 +93,12 @@ describe.only("GET /api/users", () => {
         });
       });
   });
+  test("404: returns a not found message when incorrect path enterred", () => {
+    return request(app)
+      .get("/api/usr")
```

Comment:
If you've already done a bad route test elsewhere, you possibly don't need another, as any no existent route will be caught. 
If this is the first time you're testing a bad route, then you'd need this.

Reply:
test removed